internal/storage/v2/grpc/tracereader_test.go

```diff
@@ -74,11 +85,60 @@ func startTestServer(t *testing.T, testServer *testServer) *grpc.ClientConn {
 }
 
 func TestTraceReader_GetTraces(t *testing.T) {
-	tr := &TraceReader{}
+	tests := []struct {
+		name           string
+		traces         func() []*jptrace.TracesData
+		expectedTraces func() []ptrace.Traces
+		expectedError  string
+	}{
+		{
+			name: "success",
```

Comment:
@yurishkuro I'm getting this error for this test case
```
            	Error:      	Received unexpected error:
            	            	received error from grpc stream: rpc error: code = Internal desc = grpc: failed to unmarshal the received message: proto: TracesData: illegal tag 0 (wire type 0)
```
Do you have any idea why?

Reply:
Make sure gogo is all registered. Ideally follow the pattern for api_v3 unit tests that spin the server.